<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerLongitudinalPETCTColorSelectionDialog</class>
 <widget class="QDialog" name="qSlicerLongitudinalPETCTColorSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="margin">
    <number>25</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="LabelSearch">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LineEditSearch">
       <property name="toolTip">
        <string>Filter the available colors by searching for a specific color name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="ListWidgetColors">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonSelect">
     <property name="toolTip">
      <string>Apply the color selection from the list</string>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qSlicerLongitudinalPETCTModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonSelect</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerLongitudinalPETCTColorSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ListWidgetColors</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>qSlicerLongitudinalPETCTColorSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
